tests/lib/sprintf/src/main.c

```diff
@@ -729,42 +728,23 @@ void test_sprintf_string(void)
 		     "sprintf(%%s) of REALLY_LONG_STRING doesn't match!\n");
 }
 
+
 /**
  *
  * @brief Test print function
  *
  * @see fprintf(), printf().
  *
  */
-
 void test_print(void)
 {
 	int ret, i = 3;
-	FILE *p = NULL;
 
 	ret = fprintf(stdout,  "%d", i);
 	zassert_equal(ret, 1, "fprintf failed!");
 
```

Comment:
In this function, there are two part of code segment. The first one is testing fprintf with file pointer stdout , the second one is test printf. So left the testcase name unchanged because fprintf with stdout is equal to printf.

Reply:
Ok, got it.